web_programming/instagram_crawler.py

```diff
@@ -0,0 +1,176 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+import requests
+from bs4 import BeautifulSoup
+import json
+
+headers = \
```

Comment:
No backslashes please in Python code.  See PEP8.  The problem with backslashes is that whitespace to the right of the backslash breaks the script on a change that is invisible to the reader.

You can run your code through psf/black to autofix that issue as discussed in CONTRIBUTING.md